process/block/displayBlock.go

```diff
@@ -189,27 +189,30 @@ func (txc *transactionCounter) displayTxBlockBody(
 ) []*display.LineData {
 	currentBlockTxs := 0
 
-	miniBlockHeaders := header.GetMiniBlockHeadersHashes()
+	miniBlockHeaders := header.GetMiniBlockHeaderHandlers()
 	for i := 0; i < len(body.MiniBlocks); i++ {
 		miniBlock := body.MiniBlocks[i]
 
-		scheduledModeInMiniBlockHeader, _ := process.IsScheduledMode(header, &block.Body{MiniBlocks: []*block.MiniBlock{miniBlock}}, txc.hasher, txc.marshalizer)
-		scheduledModeInMiniBlock := miniBlock.IsScheduledMiniBlock()
-
-		executionTypeInMiniBlockHeaderStr := ""
-		if scheduledModeInMiniBlockHeader {
-			executionTypeInMiniBlockHeaderStr = common.ScheduledMode + "_"
+		processingTypeInMiniBlockHeaderStr := ""
+		if len(miniBlockHeaders) > i {
```

Comment:
can you create a method for getting the processing type as string?

Reply:
Done